test/integration/test_i_aws_encryption_sdk_cli.py

```diff
@@ -72,6 +72,28 @@ def test_encrypt_with_metadata_output_write_to_file(tmpdir):
     assert output_metadata['output'] == str(ciphertext)
 
 
+@pytest.mark.skipif(not _should_run_tests(), reason='Integration tests disabled. See test/integration/README.rst')
+def test_encrypt_with_metdata_full_file_path(tmpdir):
+    plaintext = tmpdir.join('source_plaintext')
```

Comment:
Minor: This test would be a bit easier to read if these paths were all suffixed with `_output` or `_file` or `_path` or something, to make it more immediately clear that's what's being compared in the asserts.